src/containerapp/azext_containerapp/_clients.py

```diff
@@ -1281,6 +1287,16 @@ def list_files(cls, cmd, identifier, path, session_pool_endpoint):
 
         r = send_raw_request(cmd.cli_ctx, "GET", request_url, resource=SESSION_RESOURCE)
         return r.json()
+    
```

Comment:
CI style failed
src/containerapp/azext_containerapp/_clients.py:1290:0: C0303: Trailing whitespace (trailing-whitespace)

remove the whitespece

Reply:
Thanks, fixed